Composer/packages/client/src/router.tsx

```diff
@@ -50,7 +52,7 @@ const Routes = props => {
             <Publish path="publish/:targetName" />
             <Skills path="skills/*" />
           </ProjectRouter>
-          <Home path="home" />
+          <BotCreationFlowRouter path="/home/*"></BotCreationFlowRouter>
```

Comment:
Could this be something like:

```ts
<Home path="home" />
<BotCreationFlowRouter path="projects" />
```

And then in `BotCreationFlowRouter`:

```ts
          <CreateOptions path="create" />
          <DefineConversation path="create/:templateId" />
          <DefineConversation path=":projectId/save" />
          <OpenProject path="/open" />
```

Reply:
Just leaving a note here as a follow up of our convo on Teams
Since all the modals are overlayed on top of Home component. The above flow would mean rerendering of home component when we land on the home page and then an another reload when we go into the Create step wizard flow. We decided that 
<BotCreationFlowRouter path="projects/*" />
   <CreateOptions path="create" />
          <DefineConversation path="create/:templateId" />
          <DefineConversation path=":projectId/save" />
          <OpenProject path="/open" />
<BotCreationFlowRouter path="home" />
will prevent reloading of home component